.github/workflows/build_deploy_exm_fe.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy EXM FE
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '14'
+
+    - name: Install dependencies
+      run: npm install
+      working-directory: expenses-manager-frontend/src/main/frontend
+
+    - name: Build Angular application
+      run: npm run build --prod
```

Comment:
`--prod` e deprecated (vezi [aici](https://v13.angular.io/cli/build)). Poti folosi `--configuration production` sau chiar nimic avand in vedere ca by default configul selectat e cel de production. (vezi [aici](https://github.com/EndavaCollab/bvdapprenticeship2024-expenses-manager/blob/901f8364584ee1418b69b057186b267c109d2c10/expenses-manager-frontend/src/main/frontend/angular.json#L66))